src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -117,6 +148,7 @@ public class RobotContainer {
 
   public RobotContainer() {
     {
+      SensorFactory.configureCamera();
```

Comment:
This is the only line that could conceivably do harm. Has the robot been tested (e.g. auto, teleop driving, intake, and shooting) with it in place? If yes, I recommend approving the PR.

If not, comment it out and I will recommend approving the PR. Then at a worksession, uncomment it, test everything and if it works create another PR just to uncomment it.